estate/models/estate_property_type.py

```diff
@@ -8,8 +8,26 @@ class EstatePropertyType(models.Model):
     # ---------------------------------------- Private Attributes ---------------------------------
     _name = "estate.property.type"
     _description = "Real Estate Property Type"
+    _order = "sequence, name"  # asc or desc, by default asc
+    _sql_constraints = [
+        ("check_name", "UNIQUE(name)", "The name must be unique"),
+    ]
 
     # --------------------------------------- Fields Declaration ----------------------------------
 
     # Basic
     name = fields.Char(required=True)
+    sequence = fields.Integer(default=1, help="To order stages. Lower is better.")
+
+    # Relational (for inline view)
+    property_ids = fields.One2many("estate.property", "property_type_id", string="Properties")
+    offer_ids = fields.One2many("estate.property.offer", "property_type_id", string="Offer")
```

Comment:
```suggestion
    offer_ids = fields.One2many("estate.property.offer", "property_type_id")
```
I think the argument string it isn't necessary, could you please check functionally if the field appears as `Offer?`